README.md

```diff
@@ -1,23 +1,8 @@
 # SignsUi
 
-To start your Phoenix server:
+To get up and running locally:
 
-  * Install dependencies with `mix deps.get`
-  * Install `yarn` with `npm install -g yarn`
-  * Install JS dependencies with `pushd assets && yarn install && popd`
-  * Start Phoenix endpoint with `NODE_ENV=development mix run --no-halt` (substituting whatever username/password you want)
-
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
-
-To get updates on the signs locally, run `realtime_signs` with a `SIGN_UI_URL` of the local url of `signs_ui`.
-See that `README` for more details.
-
-Ready to run in production? Please [check our deployment guides](http://www.phoenixframework.org/docs/deployment).
-
-## Learn more
-
-  * Official website: http://www.phoenixframework.org/
-  * Guides: http://phoenixframework.org/docs/overview
-  * Docs: https://hexdocs.pm/phoenix
-  * Mailing list: http://groups.google.com/group/phoenix-talk
-  * Source: https://github.com/phoenixframework/phoenix
+* Install [asdf](https://github.com/asdf-vm/asdf) and the plug-ins for elixir, erlang, and nodejs.
+* Use asdf to install the proper versions of our languages: `asdf install`
+* `mix test` from repo root should now work (you should see "XYZ tests, 0 failures" at the bottom of the output)
```

Comment:
You would have to run `mix deps.get` first. Also, we seem to have omitted how to start the app itself?